workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -194,3 +152,29 @@ def create_figure(self) -> None:
         self.result.svg = img_data.getvalue()  # this is svg data
         logging.debug("Successful SVG figure creation")
         plt.close("all")
+
+    def check_corner_cases(self) -> bool:
+        """Check corner cases
+
+        If corner case is present set result description.
+
+        Returns
+            bool: Return True if no corner case present. False otherwise.
```

Comment:
I find this a bit counter intuitive. You check for a certain condition (e.g. if this is a edge case) and return True if it is not present. I would have expected that you return True if it is a corner case.

Or did you have a reason to do it the other way round?

Reply:
Yes I understand this. I changed the behavior of the function to return a string with the description of the edge case if present. I think now it is easier to understand.